### DISCUSSION TITLE
Ferret requests are concurrent or sequential

### DISCUSSION THREAD

**Discussion by rahulsingh303** [MontFerret/ferret#700] [Q&A] [answered]
Hi,

In Ferret when we are scraping data in a FOR loop , is it doing sequentially or is it concurrent ? For eg, in this code https://github.com/MontFerret/ferret/blob/b3adea3622c6d103515efa1edd48d41a2d566073/examples/dynamic-page.fql , when we are scraping artists and tracks , is Ferret going through the list sequentially or concurrently ?

-Rahul

**Comment by ziflex** [comment, marked as answer]
Hey, Rahul!
Ferret processes everything sequentially. The language does not support parallelism (yet?).
